### ISSUE TITLE
FromInteger Construction

### ISSUE THREAD

**Issue by CMCDragonkai** [chdemko/php-bitarray#1]
How come fromInteger doesn't do anything except create a bitarray with integer as the size? I thought have thought it would be equivalent to a `decbin` style construction. That is fromInteger(2) should give back the bitarray `10` and fromInteger(4) should give `100`... etc.

Would be great if it has a size designation as well, so one could easily create constants representing options in classes like:

```
class A {
    const ERROR_A = BitArray::fromInteger(1, 4);
    const ERROR_B = BitArray::fromInteger(2, 4);
    const ERROR_C = BitArray::fromInteger(4, 4);
    const ERROR_D = BitArray::fromInteger(8, 4);
}
```

Although the above seems a bit verbose. I currently just rely on the normal PHP ints.

**Comment by chdemko** [comment]
It's difficult now to change the API, what do you think to create instead a

~~~php
BitArray::fromBits(int, size)
~~~

method?

**Comment by CMCDragonkai** [comment]
Yea that might be right. But `fromBits` seems wrong. What about `fromDec`?

**Comment by chdemko** [comment]
Good idea, will change that in september (currently on holydays...)

**Comment by chdemko** [comment]
Fixed in 1.2.0
